### PR TITLE
remove jquery

### PR DESCRIPTION
**Pull request by jinpark**
Done-y done!

### CODE REVIEW

File: ad-markers.js [hunk 1]
```diff
@@ -67,9 +65,9 @@
         function addMarkers(newMarkers) {
 
             // create the adMarkers
-            $.each(newMarkers, function (index, marker) {
+            newMarkers.forEach(function (index, marker) {
                 marker.key = generateUUID();
-                videoWrapper.find('.vjs-progress-control').append(createMarkerDiv(marker));
+                videoWrapper.$('.vjs-progress-control').appendContent(createMarkerDiv(marker));
```
Comment (davidjamesherzog) [inline ad-markers.js]: Is this `$` still a part of jQuery?

Comment (jinpark) [inline ad-markers.js]: Nope, its part of videojs http://docs.videojs.com/docs/api/component.html#Methods$

**Comment by davidjamesherzog** [comment]
LGTM

**Comment by jinpark** [comment]
@davidjamesherzog can you take another look? Fixed all of my dumb mistakes

File: ad-markers.js [hunk 2]
```diff
@@ -94,22 +92,23 @@
...
-                markerDiv = $("<div class='vjs-admarker vjs-admarker-announcer'></div>");
+                markerDiv = document.createElement('div');
+                markerDiv.className += 'vjs-admarker';
+                markerDiv.className += ' vjs-admarker-announcer';
             }
             else {
-                markerDiv = $("<div class='vjs-admarker'></div>");
+                markerDiv = document.createElement('div');
+                markerDiv.className += 'vjs-admarker';
             }
```
Comment (davidjamesherzog) [inline ad-markers.js]: Probably cleaner to do this:

```
markerDiv = document.createElement('div');
markerDiv.className += 'vjs-admarker';
if (false !== marker.loader) {
  markerDiv.className += ' vjs-admarker-announcer';
}
```

**Comment by davidjamesherzog** [comment]
Don't forget to remove the jquery references from the sample.html and multiview.html

**Comment by jinpark** [comment]
Yup, I linked it here https://github.com/DramaFever/video-player/issues/136

File: ad-markers.js (hunk 2 above)
Comment (jinpark) [inline ad-markers.js]: Agreed. I just blindly followed what they did in the previous code :P

**Comment by davidjamesherzog** [comment]
Oh yeah, duh.  I forgot that this was in its own repo.

**Comment by davidjamesherzog** [comment]
LGTM

**Pull request merged**